packages/containers/src/Actions/Actions.connect.js

```diff
@@ -1,5 +1,5 @@
 import { cmfConnect } from '@talend/react-cmf';
-import { Actions } from '@talend/react-components';
+import { Actions } from '@talend/react-components/lib/Actions';
```

Comment:
```suggestion
import Actions from '@talend/react-components/lib/Actions';
```
no?  https://github.com/Talend/ui/blob/master/packages/containers/src/Actions/index.js#L3

Reply:
i was wrong sorry, you're good : [https://github.com/Talend/ui/blob/master/packages/components/src/Actions/index.js#L9](https://github.com/Talend/ui/blob/master/packages/components/src/Actions/index.js#L9)